src/locales/en-US/commands.json

```diff
@@ -499,114 +500,114 @@
+    "commandDescription": "Converts your text into binary.",
     "commandUsage": "<text>",
-    "missingText": "You need to give me some text or binary",
+    "missingText": "You have to give me some text or binary!",
     "textToBinary": "In Binary",
     "binaryToText": "In Text",
-    "tooLongTitle": "Your text is longer than 200 characters",
-    "tooLongDescription": "Try shortening your text down or separate it into multiple commands"
+    "tooLongTitle": "Your text is longer than 200 characters!",
+    "tooLongDescription": "Try shortening your text or separate it into multiple commands!"
   },
   "cowsay": {
-    "commandDescription": "Turns your text into cowsay",
-    "noText": "You need to give me a sentence!"
+    "commandDescription": "Turns your text into cowsay.",
+    "noText": "You have to give me some text!"
   },
   "inspirobot": {
-    "commandDescription": "Shows a random InspiroBot quote",
+    "commandDescription": "Shows a random InspiroBot quote.",
     "quote": "Here's your InspiroBot quote!"
   },
   "cat": {
     "commandDescription": "Fetches a random cat and displays it. Nya~",
     "hereIsYourCat": "Here's your cat!"
   },
   "i18n": {
-    "commandDescription": "Shows the link to Switchblade translation page",
+    "commandDescription": "Shows the link to Switchblade's translation page.",
     "translateMe": "**[Translate me to your language](https://translate.switchblade.xyz)**"
   },
   "beatsaver": {
-    "commandDescription": "Searches for a Beatmap on Beat Saver",
+    "commandDescription": "Searches for a Beatmap on Beat Saver.",
     "commandUsage": "<beatmap name, song, id, author or uploader>",
     "download": "Download",
     "details": "Details",
-    "noQuery": "You need to give me a map name, song, id, author or uploader.",
+    "noQuery": "You have to give me a map name, song, author, id or an uploader.",
     "notFound": "No Beatmaps were found."
   },
   "favcolor": {
-    "commandDescription": "Set a background color to your profile",
+    "commandDescription": "Sets a background color on your profile.",
     "commandUsage": "<#hexcode>",
     "changedSuccessfully": "Profile color changed to {{hexcode}}."
   },
   "rep": {
-    "commandDescription": "Gives a reputation point to someone",
+    "commandDescription": "Gives a reputation point to someone.",
     "commandUsage": "<user>",
-    "noMention": "You need to mention someone",
-    "reputationPoint": "You gave a reputation point to {{user}}",
-    "againIn": "You can do it again in **{{time}}**",
-    "alreadyGave": "You already gave a reputation point to someone today",
-    "repYourself": "You can't give a reputation point to yourself"
+    "noMention": "You have to mention someone!",
+    "reputationPoint": "You gave a reputation point to {{user}}.",
+    "againIn": "You already gave a reputation point to someone today. You can do it again in **{{time}}**.",
+    "alreadyGave": "Already gave reputation",
```

Comment:
This could have punctuation as well.

Reply:
`"alreadyGave"` is an embed title, and recently we issues a bug, with markdown in embed titles on mobile.